C7/Game.cs

```diff
@@ -395,6 +398,8 @@ public override void _UnhandledInput(InputEvent @event)
 							new RightClickTileMenu(this, tile).Open(eventMouseButton.Position);
 
 						string yield = tile.YieldString(controller);
+						//These GD.Print statements are debugging prints for developers to see info about the tile
+						//For now I'm leaving them as GD.Print.  Could revisit this later.
```

Comment:
In the future we could extend logging to have something like a `log.Console` method that wraps a `GD.Print` call but can be configured to no-op in release builds

Reply:
I like the idea!  I'm a bit Serilog'ed out right now personally, but feel free to write up the idea somewhere where it'll be found again later (which means probably outside these PR comments).